client/signup/steps/site-options/site-options.tsx

```diff
@@ -0,0 +1,75 @@
+import { Button } from '@automattic/components';
+import { Icon } from '@wordpress/icons';
+import { localize, LocalizeProps } from 'i18n-calypso';
+import React from 'react';
+import FormFieldset from 'calypso/components/forms/form-fieldset';
+import FormLabel from 'calypso/components/forms/form-label';
+import FormSettingExplanation from 'calypso/components/forms/form-setting-explanation';
+import FormInput from 'calypso/components/forms/form-text-input';
+import { tip } from './icons';
+import './site-options.scss';
+
+interface FormElements extends HTMLFormControlsCollection {
+	siteTitle: HTMLInputElement;
+	tagline: HTMLInputElement;
+}
+
+interface SiteOptionFormElement extends HTMLFormElement {
+	readonly elements: FormElements;
+}
+
+interface Props {
+	defaultSiteTitle: string;
+	defaultTagline: string;
+	onSubmit: ( siteTitle: string, tagline: string ) => void;
+	translate: LocalizeProps[ 'translate' ];
+}
+
+const SiteOptions: React.FC< Props > = ( {
+	defaultSiteTitle,
+	defaultTagline,
+	onSubmit,
+	translate,
+} ) => {
+	const handleSubmit = ( event: React.FormEvent< SiteOptionFormElement > ) => {
+		event.preventDefault();
+		const { siteTitle, tagline } = event.currentTarget.elements;
+		onSubmit( siteTitle.value, tagline.value );
+	};
+
+	return (
+		<form className="site-options__form" onSubmit={ handleSubmit }>
+			<FormFieldset className="site-options__form-fieldset">
+				<FormLabel htmlFor="siteTitle" optional>
+					{ translate( 'Blog name' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **40** times:
`translate( 'Blog Name:' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/feat/give-your-blog-name)